hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -141,9 +141,6 @@ public static void init() throws Exception {
     testFile.getParentFile().mkdirs();
     testFile.createNewFile();
 
```

Comment:
is there a reason we want to initialize for every test now? ( init is annotated as @beforeclass. )

Reply:
Thanks for the details. I agree on real-world case. I asked because to know why we are creating now and why it was not a problem before. I understand now. I am ok.